controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaInterceptor.java

```diff
@@ -86,36 +82,22 @@
         return this.handler.authorize(resource, AUTH_CONTEXT_TOKEN.get());
     }
 
-    public static String retrieveDelegationToken(String tokenSigningKey) {
```

Comment:
This bullet point in the PR description refers to this method:

> retrieveDelegationToken - which is supposed to be called from a grpc context.

but given that you are getting rid of this method and by context, I suspect that you are referring to this other method instead:

https://github.com/pravega/pravega/pull/2825/files#diff-1715d2b863069928c3a29b043e89efd0R396

If this is right, could you fix the description, please?

Reply:
Fixed the description.